keyboards/dz60/keymaps/pinpox/rules.mk

```
@@ -0,0 +1,13 @@
+
+# Build Options
+# override the options specified in dz60/rules.mk
+BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
+# MOUSEKEY_ENABLE = yes       # Mouse keys
+# EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+# KEY_LOCK_ENABLE = yes       # Enables use of KC_LOCK
```

This can be reduced to just:
```suggestion
BOOTMAGIC_ENABLE = lite
```
Since the rest of the uncommented options here are the same as the keyboard-level rules.mk.